### ISSUE TITLE
map vault filter only up to 12 players

### ISSUE THREAD

**Issue by CookieNoob** [FAForever/client#737]
**I experienced...** I wanted to filter the maps in the map vault for maps with more than 14 players

**how to reproduce** put a filter in the map vault search

**I expected...** the filter to go up to 16 (max player count in the lobby)

**Where the bug is / What should be done** increase the max number of players in the filter option

![grafik](https://cloud.githubusercontent.com/assets/21953792/26458437/8adbe804-4173-11e7-960d-3181dc6a99c5.png)

**Comment by GrotheFAF** [comment]
filter is part of the used webpage, nothing can be done about that in the client.

**Comment by duk3luk3** [comment]
We need to replace the map vault with something that uses the API directly.

**Comment by Wesmania** [comment]
Which in turn requires the API to be implemented in the first place.

**Comment by Brutus5000** [comment]
No. Just make calls against the public API parts (no oauth authorization required for maps).
Obtain the map list with a JSON API library via
api.dev.faforever.com/data/map?include=versions&filter[mapVersion]=hidden==False

**Comment by duk3luk3** [comment]
Wesmania means accessing the API needs to be implemented in the client, which we're currently working on.
